java/connector-node/risingwave-sink-cassandra/src/main/java/com/risingwave/connector/CassandraSink.java

```diff
@@ -122,6 +124,7 @@ private void write_append_only(Iterator<SinkRow> rows) {
                             .withDescription("Unknown operation: " + op)
                             .asRuntimeException();
             }
+            tryCommit();
```

Comment:
This optimization of cassandra sink can be put in a separate PR and we can paste improvement gained from the optimization in the PR description.

Reply:
This is a bug, fixed in this pr by the way, the exact description has been added to the pr description